src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,31 @@
+package lotto.controller;
+
+import lotto.domain.GameResult;
+import lotto.domain.LottoTicket;
+import lotto.domain.LottoTickets;
+import lotto.domain.Money;
+import lotto.service.LottoService;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoController {
+    InputView inputView;
+    ResultView resultView;
+    LottoService lottoService;
+
+    public LottoController(InputView inputView, ResultView resultView) {
+        this.lottoService = new LottoService();
+        this.inputView = inputView;
+        this.resultView = resultView;
+    }
+
+    public void run() {
+        Money inputMoney = new Money(inputView.inputMoney());
+        LottoTickets lottoTickets = lottoService.buyLottoTickets(inputMoney);
+        ResultView.printBuyResult(lottoTickets.count(), lottoTickets.toResultString());
+        LottoTicket winningLottoTicket = LottoTickets.fromString(inputView.inputWinningNumber());
+        GameResult gameResult = lottoService.getGameResult(lottoTickets, winningLottoTicket);
+        ResultView.printGameResult(gameResult.toString());
+        ResultView.printEarningRatio(inputMoney, new Money(gameResult.getPrize()));
```

Comment:
개행을 문맥을 분리하는데 사용하면 가독성을 높여준답니다 :)
```suggestion
        // 로또 구입
        Money inputMoney = new Money(inputView.inputMoney());
        LottoTickets lottoTickets = lottoService.buyLottoTickets(inputMoney);
        ResultView.printBuyResult(lottoTickets.count(), lottoTickets.toResultString());
        
        // 당첨 번호 입력
        LottoTicket winningLottoTicket = LottoTickets.fromString(inputView.inputWinningNumber());
        
        // 결과 출력
        GameResult gameResult = lottoService.getGameResult(lottoTickets, winningLottoTicket);
        ResultView.printGameResult(gameResult.toString());
        ResultView.printEarningRatio(inputMoney, new Money(gameResult.getPrize()));
```